script/Common/00_00.txt

```diff
@@ -1329,12 +1336,13 @@ I treated you as a fellow human and gave you my opinion, got angry and slapped y
 [sha:973ef4df7d37b734d80b14bc7020add8e3947e22]{
 -- Page 96, Offset 361.
 -- その言葉は、本当に怖かった。
-Those words were really scary.
+Those words are really scary.
 }
 [sha:8497c7420462ffb56458d56b2b77fd2c1d8bad6a]{
 -- Page 96, Offset 362.
 -- 自分が殺されていたかもしれない、という“もしも”ではなくて、この人に、そんなことをさせてしまうことが、あまりにも悲しかったのだ。
-I wasn't sad because I might've been killed, but because I didn't want this person to have to do such a thing to me.
+// try to link this with the previous line like it was intended.
+Not because of the possibility that she might have killed me, but because it's just too sad that I'd done something to make her do so.
```

Comment:
Up to the comma is fine, but _make her do so_ makes it sounds like she actually _did_ kill him
> ...killed me, but because I was sad that I'd forced her to come so close to doing so.

Reply:
> ...killed me, but because _of how sad it was_ that I'd forced her to come so close to doing so.
Slight change, I think this is closer to intent